src/Services/SiteManager/Client.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Bonnier\WP\ContentHub\Editor\Services\SiteManager;
+
+/**
+ * Class Client
+ *
+ * @package \Bonnier\WP\ContentHub\Editor\Services\SiteManager
+ */
+class Client extends \GuzzleHttp\Client
+{
+    protected static $instance = null;
+
+    /**
+     * Client constructor.
+     */
+    public function __construct()
+    {
+        parent::__construct([
+            'base_uri' => getenv('SITE_MANAGER_HOST'),
```

Comment:
Nej jeg besluttede mig for at rykke det til en env setting, synes ikke det er noget som folk med adgang til WP backend'en skal kunne ændre.